### ISSUE TITLE
pandas version 1.0.0 causes error

### ISSUE THREAD

**Issue by dwr-psandhu** [CADWRDeltaModeling/pyhecdss#15]
* pyhecdss version:
* Python version:
* Operating System:

### Description
Installing pyhecdss installs latest version of pandas i.e. 1.0.0 which causes an error on import

### What I Did
import pyhecdss

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\psandhu\AppData\Local\Continuum\anaconda3\envs\test1\lib\site-packages\pyhecdss\__init__.py", line 4, in <module>
    from .pyhecdss import *
  File "C:\Users\psandhu\AppData\Local\Continuum\anaconda3\envs\test1\lib\site-packages\pyhecdss\pyhecdss.py", line 34, in <module>
    class DSSFile:
  File "C:\Users\psandhu\AppData\Local\Continuum\anaconda3\envs\test1\lib\site-packages\pyhecdss\pyhecdss.py", line 245, in DSSFile
    def _number_between(startDateStr, endDateStr, delta=pd.to_timedelta(1, 'Y')):
  File "C:\Users\psandhu\AppData\Local\Continuum\anaconda3\envs\test1\lib\site-packages\pandas\core\tools\timedeltas.py", line 87, in to_timedelta
    "Units 'M' and 'Y' are no longer supported, as they do not "
ValueError: Units 'M' and 'Y' are no longer supported, as they do not represent unambiguous timedelta values durations.
```

**Comment by kjnam** [comment]
FYI, I can reproduce the same issue.

**Comment by dwr-psandhu** [comment]
fixed by commit  11d1f6102f9a89c66a6098ad8f0dee80598c4ac4
